composer.json

```diff
@@ -31,21 +31,25 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
-        "illuminate/config": "^6.0 || ^7.0 || ^8.0",
-        "illuminate/database": "^6.0 || ^7.0 || ^8.0",
-        "illuminate/support": "^6.0 || ^7.0 || ^8.0"
+        "php": "^8.0",
+        "illuminate/config": "^8.0",
+        "illuminate/database": "^8.0",
+        "illuminate/support": "^8.0"
     },
     "require-dev": {
         "ext-json": "*",
-        "orchestra/testbench": "^4.0 || ^5.0 || ^6.0",
+        "orchestra/testbench": "^6.0",
         "phpunit/phpunit": "^9.3"
     },
     "config": {
         "sort-packages": true
     },
     "extra": {
         "laravel": {
+            "aliases": {
+                "CauserResolver": "Spatie\\Activitylog\\Facades\\CauserResolver",
```

Comment:
Please remove aliases - aliases are one of the worst about Laravel and the devs using them can spend the minute to configure them their own.